Makefile

```diff
@@ -38,3 +38,7 @@ pkg: build
 # builds, packages, and installs the extension to your VS Code
 install: pkg
 	code --force --install-extension test_extension.vsix
+
+.PHONY: nix/opam-selection.nix
+nix/opam-selection.nix:
```

Comment:
Why not `nix-shell`, or even `nix`? It's a bit shorter

Reply:
Well it's named after the target it produces. It doesn't need a short name as it's not something you should run often. It regenerates the build plan from opam-repository which is something that only needs to be done to if you're updating dependencies.